ecobalyse_data/computation.py

```diff
@@ -132,13 +147,14 @@ def compute_processes_for_activities(
         )
         index += 1
 
-        process = compute_process_for_activity(
+        process, processed_activities = compute_process_for_activity(
```

Comment:
Let’s do the deduplication here. But do we really want do it? I mean, we have activities with the same name and source? If so, it’s a bug no?

Reply:
yes we have activities with the same name and source, that are not bugs. See :

`jq '.[] | select(.search | contains("Cauliflower, winter, conventional, at farm gate {FR} U"))' activities.json`


<img width="531" alt="image" src="https://github.com/user-attachments/assets/93b851bb-ac18-4d7d-b839-3ac77e80b6f2" />